module-bluetooth/Bluetooth/error_bluetooth.cpp

```diff
@@ -0,0 +1,138 @@
+#include "error_bluetooth.hpp"
+#include <module-bluetooth/lib/btstack/src/bluetooth.h>
+
+namespace bluetooth {
+const char *error_cstr(int err)
```

Comment:
Nice candidate to replace with [X-Macro](https://quuxplusone.github.io/blog/2021/02/01/x-macros/) technique

Reply:
I even use string expansion macro in updater for the very same case, but for c++ it's considered code smell, therefore not added here.
Saying that, the creation of more elaborate Xmacros ( how author called them) is a horrible mess to debug for more complex solutions :)
Here it would be pretty simple if in Bluetooth library this would be an enum - magic enum would suffice.

Reply:
You're right, `magic_enum` would be a better choice.